x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/search_or_filter.cy.ts

```diff
@@ -53,18 +55,17 @@ describe('Timeline search and filters', { tags: ['@ess', '@serverless'] }, () =>
     });
```

Comment:
line 52 the `executeTimelineSearch` method has a `force: true` that we should try to remove ([here](https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/cypress/tasks/timeline.ts#L367))